classes/kernel-lmp-fitimage.bbclass

```diff
@@ -2,15 +2,19 @@
 
 inherit kernel-fitimage
 
+# Default value for deployment filenames
+FPGA_BINARY ?= "fpga.bin"
+
 #
 # Emit the fitImage ITS configuration section
 #
 # $1 ... .its filename
 # $2 ... Linux kernel ID
 # $3 ... DTB image name
 # $4 ... ramdisk ID
-# $5 ... config ID
```

Comment:
Are we dropping config ID? Or are we calling that Setup ID now?

Reply:
Original comment was wrong, $5 is actually setup id.

Reply:
ok